src/components/chart/_chart.scss

```
@@ -7,6 +7,12 @@
         overflow: visible !important;
     }
 
+    &__container:focus {
+        box-shadow: 0 0 0 3px black !important;
```

```suggestion
        box-shadow: 0 0 0 3px var(--ons-color-black) !important;
```
We can use the var here